sdk/template/template/test/snippets.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { DefaultAzureCredential } from "@azure/identity";
+import { setLogLevel } from "@azure/logger";
+import { ConfigurationClient } from "../src";
```

Comment:
Should we change this already to `@azure/template` since it should be mapped via the tsconfig file?

Reply:
That's a good question. While the tsconfig handles the mapping at compile time, runtime support for mapping these imports is provided by dev-tool, but currently dev-tool can only host samples. I have [a PR](https://github.com/Azure/azure-sdk-for-js/pull/23852) to add support for hosting the tests to dev-tool, but it breaks some tests for reasons that I've found difficult to understand.

Being able to import from `"@azure/template"` in tests like you're suggesting is blocked on that PR.